martin/src/srv/server.rs

```diff
@@ -96,6 +104,15 @@ pub fn router(cfg: &mut web::ServiceConfig) {
 
     #[cfg(feature = "fonts")]
     cfg.service(crate::srv::fonts::get_font);
+
+    #[cfg(feature = "webui")]
+    cfg.service(actix_web_static_files::ResourceFiles::new(
```

Comment:
there is a subtle bug here - compiled files go into `/assets/...` - but the word `assets` could also be the name of a tile source. Is there a way we can make the root `/index.html` use `/_/assets/...` path instead?  The `_` is a reserved keyword, so we should put everything non-stable there.